apps/mobile/src/components/Community/CommunityMetadataFormBottomSheet.tsx

```diff
@@ -0,0 +1,158 @@
+import { useBottomSheetDynamicSnapPoints } from '@gorhom/bottom-sheet';
+import clsx from 'clsx';
+import { ForwardedRef, forwardRef, useCallback, useRef, useState } from 'react';
+import { TextInput, View } from 'react-native';
+import { graphql, useFragment } from 'react-relay';
+import colors from 'shared/theme/colors';
+
+import { useToastActions } from '~/contexts/ToastContext';
+import { env } from '~/env/runtime';
+import { CommunityMetadataFormBottomSheetFragment$key } from '~/generated/CommunityMetadataFormBottomSheetFragment.graphql';
+import { contexts } from '~/shared/analytics/constants';
+
+import { Button } from '../Button';
+import {
+  GalleryBottomSheetModal,
+  GalleryBottomSheetModalType,
+} from '../GalleryBottomSheet/GalleryBottomSheetModal';
+import { useSafeAreaPadding } from '../SafeAreaViewWithPadding';
+import { Typography } from '../Typography';
+
+const SNAP_POINTS = ['CONTENT_HEIGHT'];
+
+type Props = {
+  communityRef: CommunityMetadataFormBottomSheetFragment$key;
+};
+
+function CommunityMetadataFormBottomSheet(
+  { communityRef }: Props,
+  ref: ForwardedRef<GalleryBottomSheetModalType>
+) {
+  const community = useFragment(
+    graphql`
+      fragment CommunityMetadataFormBottomSheetFragment on Community {
+        dbid
+        name
+      }
+    `,
+    communityRef
+  );
+
+  const { bottom } = useSafeAreaPadding();
+  const { pushToast } = useToastActions();
+
+  const bottomSheetRef = useRef<GalleryBottomSheetModalType | null>(null);
+
+  const { animatedHandleHeight, animatedSnapPoints, animatedContentHeight, handleContentLayout } =
+    useBottomSheetDynamicSnapPoints(SNAP_POINTS);
+
+  const [message, setMessage] = useState<string>('');
+  const [status, setStatus] = useState<'SUBMITTING' | 'SUCCESS' | 'ERROR' | undefined>();
+
+  const handleSubmitPress = useCallback(async () => {
+    setStatus('SUBMITTING');
+
+    try {
+      const response = await fetch(
+        `https://formspree.io/f/${env.EXPO_PUBLIC_FORMSPEE_REQUEST_COLLECTION_ID}`,
+        {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            communityId: community.dbid,
+            communityName: community.name,
+            message,
```

Comment:
same comment as web - could you include the submitter's user dbid 🙏

Reply:
Added!